pkg/render/intrusion_detection.go

```diff
@@ -1385,12 +1078,12 @@ func (c *intrusionDetectionComponent) intrusionDetectionControllerAllowTigeraPol
 		egressRules = append(egressRules, v3.Rule{
 			Action:      v3.Allow,
 			Protocol:    &networkpolicy.TCPProtocol,
-			Destination: networkpolicy.DefaultHelper().ESGatewayEntityRule(),
+			Destination: networkpolicy.Helper(c.cfg.Tenant.MultiTenant(), c.cfg.Namespace).ESGatewayEntityRule(),
```

Comment:
Instead of instantiating a new instance of a helper per rule we can just instantiate a single one and re-use it for each rule. Probably paranoid but may avoid bugs if the rules are all using the same helper vs unique instances.